python/upt-cpan/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                upt-cpan
+version             0.4
+revision            0
+
+maintainers         nomaintainer
+description         CPAN frontend for upt
+long_description    ${description}
+
+platforms           darwin
+supported_archs     noarch
+
+homepage            https://framagit.org/upt/upt-cpan
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+license             3-clause BSD License
+checksums           sha256  47ebc37bf4510867858eec52e2d23221bc5a0f061a057b41892cb96b8e3def14 \
+                    rmd160  418532c5abb388d4ddcb050610944266ba1f456c \
+                    size    5091
+                                
+python.versions     37
+
+
+depends_build-append \
+                    port:py${python.version}-setuptools
+
+depends_lib-append \
+                    port:py${python.version}-requests \
+                    port:upt
+
```

Comment:
please install the files in ```post-destroot``` (see above); additionally please add the code to enable tests. You can take a look at other Python ports for examples on how to do this or check [the guide](https://guide.macports.org/#reference.phases.test).